src/components/EventModal/Collect/index.vue

```diff
@@ -199,14 +229,25 @@
 
       <section v-if="paymentMethod === undefined">
         <Label
-          v-if="enableStripe"
+          v-if="!nftIsFree && enableStripe"
           class="text-like-green"
           preset="h5"
           align="center"
           :text="$t('nft_collect_modal_subtitle_select_collect_method')"
         />
-        <ul v-if="nftIsFree" class="mt-[16px] flex flex-col gap-[16px] mx-auto max-w-[320px] w-full">
-          <li >
+        <ButtonV2
+          v-if="nftIsMidAutumnStyle && nftIsFree"
+          class="mx-auto mt-[3rem]"
+          :text="$t('nft_collect_modal_method_free')"
+          :is-disabled="mintedFreeNFT"
+          preset="secondary"
+          @click="handleSelectPaymentMethod('free')"
+        />
+        <ul
+          v-else-if="nftIsFree"
+          class="mt-[16px] flex flex-col gap-[16px] mx-auto max-w-[320px] w-full"
+        >
```

Comment:
Seems more readable if we use <template v-if="nftIsFree"> to wrap these two, but this would also be fine